CMakeLists.txt

```diff
@@ -61,3 +61,18 @@ set(PYTHON_LIBRARIES "/usr/lib/x86_64-linux-gnu/libpython3.5m.so")
 include_directories(${PYTHON_INCLUDE_DIRS})
 include_directories(${PYTHON_DIRECTORIES})
 target_link_libraries(JasmineGraph ${PYTHON_LIBRARIES})
+target_compile_options(JasmineGraph PRIVATE -std=c++11)
```

Comment:
We need not commit this file since there are no significant changes. Lets not add these changes into the repo.

Reply:
fixed in https://github.com/miyurud/jasminegraph/pull/104/commits/23322798c6fda122e79f935250c2cea59b4f4dbc

Reply:
Done